java-util/src/main/java/io/druid/java/util/common/granularity/NoneGranularity.java

```diff
@@ -25,16 +25,9 @@
 /**
  * NoneGranularity does not bucket data
  */
-public final class NoneGranularity extends Granularity
+public class NoneGranularity extends Granularity
 {
-  private static final NoneGranularity INSTANCE = new NoneGranularity();
-
-  private NoneGranularity() {}
-
-  public static NoneGranularity getInstance()
-  {
-    return INSTANCE;
-  }
+  public NoneGranularity() {}
```

Comment:
Not sure.  I told him to remove the singleton because singletons are just generally not great a great code pattern.  The singleton-ness of the object is ensured by the "user" reusing the same object over and over again.  There is no reason a class should enforce its own singleton-ness.